app/controllers/qa/issues_controller.rb

```diff
@@ -12,6 +12,16 @@ def index
 
   def show; end
 
+  def edit
+    @form_cancel_path = if params[:redirect_to] && URI.parse(params[:redirect_to]).relative?
+      params[:redirect_to]
+    else
+      project_issue_path(current_project, @issue)
```

Comment:
What's the use case for redirecting to issues#show from QA? I'm thinking `@form_cancel_path` will always be QA::issues#show, correct?

Reply:
yep, I missed updating this when I moved it over from IssuesController.

Updated now to `@form_cancel_path = project_issue_qa_path(current_project, @issue)`